namespace-package/README.md

```diff
@@ -0,0 +1,36 @@
+# What's a Namespace Package?
+
+Materials for the corresponding Real Python article.
+
+You have the Hello World example, and the source code for the Snake Shop Corporation example that's hosted on PyPI. Aditionally, as an extra, you have the code to generate the fake data used.
+
+You need to install every subpackage to get it working properly.
+
+## Examples
+
+In this repository you'll find four different examples showing ways namespace packages work or can be used.
+
+### Hello World
```

Comment:
@iansedano @gahjelle Is Hello World still present in the repository? I don't see it, and it's not mentioned in the tutorial.